builtin/providers/aws/structure.go

```diff
@@ -695,7 +695,10 @@ func flattenElastiCacheParameters(list []*elasticache.Parameter) []map[string]in
 func expandStringList(configured []interface{}) []*string {
 	vs := make([]*string, 0, len(configured))
 	for _, v := range configured {
-		vs = append(vs, aws.String(v.(string)))
+		val, ok := v.(string)
+		if ok && val != "" {
```

Comment:
Are there tests failing that you suspect this will fix? 
`expandStringList` is used a lot, so this change may be far reaching. It seems innocuous though.